developer-handbook/github-process/post-code-review.md

```diff
@@ -0,0 +1,13 @@
+# Post Code Review
+
+Your Pull Request was reviewed by a colleague (or a few colleagues, which is great). What's left to do now is
+
+## Address it
+- The "keep your comments friendly" rule applies here too! We all have the best intentions for our codebases.
+- React or reply to every comment that was left, even if it's just "acknowledged" or "done", or an emoji reaction. The person that left it thought it was worth saying something, it's nice to acknowledge it.
```

Comment:
👍 Comments can be marked "resolved" to indicate that they've been addressed whether action is taken or not & responding "not addressing this here, but in a future PR" can be a valid reply if it makes sense in context.

Reply:
When It's my work being reviewed, I like to use "resolved" as a sort of checklist of things I have to do.  Particularly if there are a few identical points in different places - (e.g. "typo" ... "and here"... "same again here") where they do not (all) warrant a reply.

If the baton gets handed back to the reviewer after changes, that can be an opportunity for the reviewer to say "This one hasn't actually been resolved"

Reply:
I agree, when it's something small like a typo, it doesn't need a second opinion! And like you said, otherwise it's nice to have confirmation you got the recommended change right.

Reply:
Yes, unless it's a trivial change the original reviewer/commenter should have a chance to check out how the feedback was addressed

Reply:
I'm adding this below 

> The owner of the PR can "resolve" comments after commenting "done" if they are small things like typos. For bigger changes than that, the responsibility of resolving should be passed on the the reviewer as they can use it to confirm the changes are inline with what they'd suggested.

Does that suit?

Reply:
Added [here](https://github.com/wellcomecollection/developer-handbook/pull/1/files#diff-1e074398a9aa14bdce2e9c63512c5f37cbb8a8463c79f72c58e10483258412fbR10)